api_tests/nodes/views/test_node_contributors_list.py

```diff
@@ -159,6 +159,16 @@ def test_filtering_on_invalid_field(self):
         errors = res.json['errors']
         assert_equal(len(errors), 1)
         assert_equal(errors[0]['detail'], 'Query string contains an invalid filter.')
+    
+    def test_filtering_on_permission_field(self):
+        # regression test for changes in filter fields
+        # permission is now permissions
+        url_permission = '/{}nodes/{}/contributors/?filter[permission]=foo'.format(API_BASE, self.project._id)
```

Comment:
Ah, filtering the valid field on the unknown value just returns no results, but a success error code. Ok.